server/resources/[LOG]/webhook/server.lua

```diff
@@ -0,0 +1,101 @@
+
+local VorpCore = {}
+
+TriggerEvent("getCore", function(core)
+    VorpCore = core
+end)
+
+function send_to_splunk(data, source_type)
+    local endpointUrl = "YOUR SPLUNK LINK HERE"
+    local authToken = "YOUR SPLUNK TOKEN HERE"
+    local index = "main"
+    local sourcetype = source_type
+    local headers = {
+        ["Authorization"] = "Splunk " .. authToken,
+        ["Content-Type"] = "application/json",
+    
+      }
+      local body = json.encode({
+        index = index,
+        sourcetype = sourcetype,
+        event = data
+      })
+      PerformHttpRequest(endpointUrl, function(statusCode, responseBody, responseHeaders)
+        if statusCode ~= 200 then
+          print("Failed to send data to Splunk:", responseBody)
+        end
+      end, "POST", body, headers)
+end -- ends send_to_splunk
+
+
+
+function send_to_discord(data, channel)
+  local channel = channel
+  local discord = ""
+  if channel == "mailbox Broadcast"   then
+      --enter discord link next line for that channel
+      discord = "YOUR-FIRST-DISCORD-WEBHOOK-HERE" 
+  elseif channel == "chat" then
+      --enter discord link next line for that channel
+      discord = "YOUR-SECOND-DISCORD-WEBHOOK-HERE" 
+  elseif channel == "bank" then
+      --enter discord link next line for that channel
+      discord = "YOUR-THIRD-DISCORD-WEBHOOK-HERE" 
+  
+  end
+  local headers = {
+    ["Content-Type"] = "application/json"
+  }
+  local decoded_data = json.decode(data.data)
+  local message = decoded_data.message
+  local discord_body = json.encode({
+    username = data.char,
+    content = message,        
+  })
+
+  PerformHttpRequest(discord, function(statusCode, responseText, headers)
+    if statusCode ~= 204 then
+      print("Failed to send message to Discord webhook:", responseText)
+    end
+  end, "POST", discord_body, headers)
+end -- end of send to discord
+
+
+
+exports('send_to_splunk', send_to_splunk)
+exports('send_to_discord', send_to_discord)
+
+-- RegisterCommand("testing_splunk", function(src, args)
```

Comment:
we should consider conditionally making this function accessible using a variable like

IS_DEV_MODE: boolean